addons/viewport/src/manager.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import addons from '@storybook/addons';
+
+import { Panel } from './components/Panel';
+
+const ADDON_ID = 'storybook-addon-viewport';
+const PANEL_ID = `${ADDON_ID}/addon-panel`;
+
+function addChannel(api) {
```

Comment:
Looks like these could be arrow functions? any reason to make them function declarations?

Reply:
not a huge thing - but from what i understand chrome specifically will do a better job at optimizing named functions like this over arrow functions. happy to update to be an arrow function instead :D